### DISCUSSION TITLE
Main Dashboard organization: By Alias or Subdomains

### DISCUSSION THREAD

**Discussion by Argandov** [simple-login/app#985] [Feature Requests]
I am pretty new to SL, and I already have a mess in my main dashboard, and I need to Ctrl+F to find my alias (If I remember it correctly), so if the dashboard can get a little bit more organized (By email, domain, subdomain, or even tagging) the user experience could improve. Now I need to remember a bunch of different aliases  😢

**Comment by ntnhon** [comment]
There is a filter option which allows you to search aliases in different ways
<img width="1188" alt="Screenshot 2022-05-13 at 09 26 57" src="https://user-images.githubusercontent.com/4153453/168233260-e775be4b-df4d-4067-9033-42803eb1973c.png">
